okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -209,7 +211,11 @@ private static RemotingOkHttpClient createInternal(
         client.followRedirects(false); // We implement our own redirect logic.
 
         // SSL
-        client.sslSocketFactory(new KeepAliveSslSocketFactory(config.sslSocketFactory()), config.trustManager());
+        SSLSocketFactory sslSocketFactory = MetricRegistries.instrument(
+                config.taggedMetricRegistry(),
+                new KeepAliveSslSocketFactory(config.sslSocketFactory()),
+                serviceClass.getSimpleName());
```

Comment:
We could use `"client"` here, but I think it's helpful to differentiate our clients. We could alternatively do this wrapping ClientConfiguration creation with a SslSocketFactory instead, retaining the ServiceConfiguration name, and falling back to `"client"` when that information is unavailable. That would mean plumbing a service name through api bits.

Reply:
We'll have to use the ServiceConfiguration name to handle this in conjure-rust-runtime, but it probably doesn't matter all that much that the tag name is exactly the same between different services.